tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -108,7 +97,7 @@ def _run(self):
                                     hostname=peers['peer%s.primary_hostname' % index],
                                     state=peers['peer%s.state' % index]
                                 )
-                            peer.save()
+                            peer.save(ttl=200)
```

Comment:
the ttl value should be calculated  using local config flag `sync_interval` + 15 seconds 

check this for more details https://github.com/Tendrl/gluster-integration/pull/313